TestFoundation/TestNSString.swift

```diff
@@ -1039,8 +1039,8 @@ func checkHasPrefixHasSuffix(lhs: String, _ rhs: String, _ stack: [UInt]) -> Int
     }
 
     var failures = 0
-    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [__LINE__])
-    failures += testFailure(expectHasSuffix, lhs.hasSuffix(rhs), stack + [__LINE__])
+    failures += testFailure(expectHasPrefix, lhs.hasPrefix(rhs), stack + [#line])
```

Comment:
"This includes keeping `__FUNCTION__`, and making `#line` have the dual behavior of being a directive when it is the first token on a line, but an expression otherwise." [From SE-0028](https://github.com/apple/swift-evolution/blob/master/proposals/0028-modernizing-debug-identifiers.md)
So if `#line` is taken as expression it looks like it should work. Or i'm missing something.